### PR TITLE
Tagging tables

### PR DESCRIPTION
**Pull request by dericed**
This PR is almost entirely formatting only, though I moved some parenthetical comments such as “tags containing other tags” from section headers to full sentences.

I also remove the section of the table with “Colour coding/subjective information/subject to change or removal” since it’s out of context in a markdown context.

### CODE REVIEW

File: tagging.md [hunk 1]
```diff
@@ -34,384 +31,209 @@ Advanced-users application might let you put any tag in your file. But for the r
...
-    To store a specific day such as May 1st, 2003, you would use "2003-05-01".
-  
-* Fields that require a Float SHOULD use the "."
-    mark instead of the "," mark. To display it differently for another
-    local, applications SHOULD support auto replacement on display. Also, a thousandths
-    separator SHOULD NOT be used.
-* For currency amounts, there SHOULD only be a numeric value in the Tag. Only
-    numbers, no letters or symbols other than ".". For instance, you
-    would store "15.59" instead of "$15.59USD".
+* The fields with dates SHOULD have the following format: YYYY-MM-DD HH:MM:SS.MSS YYYY = Year, -MM = Month, -DD = Days, HH = Hours, :MM = Minutes, :SS = Seconds, :MSS = Milliseconds. To store less accuracy, you remove items starting from the right. To store only the year, you would use, "2004". To store a specific day such as May 1st, 2003, you would use "2003-05-01".
```
Comment (retokromer) [inline tagging.md]: I would say `MM = month` etc. instead of `-MM = Month` etc.
(i.e. without the leading sign and I don’t see a need for capitals here)

**Comment by dericed** [comment]
I suggest to leave the patch as format-only and follow with other patches (nudge to @retokromer) for content.

**Comment by robUx4** [comment]
I updated the PR with changes on the date. LGTM for the rest.

**Comment by retokromer** [comment]
LGTM

**Comment by dericed** [comment]
LGTM as well so merging now

**Pull request merged**